app/src/components/inputs/inputChekbox/inputCheckbox.jsx

```diff
@@ -18,56 +18,62 @@
  * You should have received a copy of the GNU General Public License
  * along with Report Portal.  If not, see <http://www.gnu.org/licenses/>.
  */
-
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import styles from './inputCheckbox.scss';
 
 const cx = classNames.bind(styles);
 
-export const InputCheckbox = ({ children, value, disabled, onChange, onFocus, onBlur }) => {
-  const squareClasses = cx({
-    square: true,
-    centered: !children,
-    checked: value,
-    disabled,
-  });
-  const handlerOnChange = (e) => {
-    onChange({ value: e.target.checked });
+export class InputCheckbox extends Component {
+  static propTypes = {
+    children: PropTypes.node,
+    value: PropTypes.bool,
+    disabled: PropTypes.bool,
+    onChange: PropTypes.func,
+    onFocus: PropTypes.func,
+    onBlur: PropTypes.func,
+  };
+  static defaultProps = {
+    children: '',
+    value: false,
+    disabled: false,
+    onChange: () => {},
+    onFocus: () => {},
+    onBlur: () => {},
+  };
+  state = {
+    checked: this.props.value,
```

Comment:
Why do you store value in the state instead of getting it from the props directly? By doing this you lose possibility to change component's value from the outside.